packages/core/src/html/CustomListIconProcessor.ts

```diff
@@ -25,9 +25,40 @@ interface IconAttributes {
   once?: boolean;
 }
 
+class TextsManager {
+  texts: string[] = [];
+  pt: number = 0;
+  size: number = 0;
+  constructor() {
+    this.texts = [];
+  }
+
+  isUsed() {
```

Comment:
```suggestion
  isEmpty() {
```

Reply:
I think isUsed is a better name; 
while the abstraction is slim, but with this name, the user do not need to think and interact with the string array under the hood, but have the feel of interacting with the manager.
Emm, how about I change this to isInUse?